rust/worker/src/execution/operators/fetch_log.rs

```diff
@@ -0,0 +1,84 @@
+use std::time::{SystemTime, SystemTimeError, UNIX_EPOCH};
```

Comment:
what is the difference between this operator and the pull logs operator that we already have?

Reply:
The logic is mostly the same, and the only difference should be the operator struct itself and its input. I'm leaving it as a separate operator because I don't want to interrupt the existing pipeline until I can fully replace it.

Reply:
I'm also wondering why this is introduced now; could you update the PR description with the difference between the two and some details on why we shouldn't update the PullLogsOperator instead? (granted this is a very small operator so maybe not a huge deal, just want to understand)

Reply:
@Sicheng-Pan explained this to me in-person, I think it's still worth updating the PR description for others